### DISCUSSION TITLE
middleware/mysql/faq/common-question/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4888] [Announcements]
# middleware/mysql/faq/common-question/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/mysql/faq/common-question/

**Comment by samzong** [comment]
“CR 创建数据库失败报错” 这个应该着重标记下，使用此方式会导致数据库被重置，数据丢失。

**Comment by windsonsea** [comment]
Fixed in PR #5274
